src/constants/messages.ts

```diff
@@ -57,6 +65,9 @@ export const SUCCESS_MESSAGES = {
   SHARE_ITEM: 'SHARE_ITEM',
```

Comment:
I'm not sure I understood why the failure messages are translated with values, while the success messages are translated as keys? IMO we should decide the naming strategy to be either label-based (all translations have label names i.e. keys), or default-based (all translations are named after the default version, i.e. the corresponding english sentence with the caveat that changes need to be repeated in each translation file).